### ISSUE TITLE
nbconvert --to html skips Markdown within HTML tags

### ISSUE THREAD

**Issue by vlbrown** [jupyter/nbconvert#1125]
nbconvert to html skips Markdown within HTML tags

e.g.
```
<blockquote>
This is a sentence containing some _italic_ and **bold**  text.
   * bullet point
   * bullet point

Here's a new paragraph.
</blockquote>
```
nbconvert --to html will bypass the conens of that blockquote section.

(Note that if you don't put in the `</blockquote>`, Markdown still renders as a blockquote (to the end of th cell) AND `nbconvert --to html` converts properly.  Apparently, `nbconvert` is triggered by the closing tag, not the opening tag.

```
{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Test notebook\n",
    "\n",
    "Here is some text\n",
    "\n",
    "<blockquote>\n",
    "This is quoted text with an _italicized part_ and some **bold text**.\n",
    "\n",
    "Here's a list\n",
    "   * item 1\n",
    "   * item 2\n",
    "   \n",
    "and some code:\n",
    "```\n",
    "print('hello, world')\n",
    "```\n",
    "</blockquote>\n",
    "\n",
    "And some more text."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
```

```
jupyter nbconvert --version
5.6.0
```

**Comment by joelostblom** [comment]
This would be convenient, especially with the bootstrap-like colored boxes in JupyterLab, that show up nicely inside the notebooks:

![image](https://user-images.githubusercontent.com/4560057/92413593-a09d1100-f105-11ea-923e-591c1b3f54a8.png)

I realize that it could be rewritten to use HTML tags, but I prefer to have it in markdown to be able to export to PDF when needed (even if the box is not included), and since I have scripts that parse the markdown headers.

**Comment by mgeier** [comment]
Yes, it would be great if `nbconvert` (and by extension `nbviewer`) could parse those contents of HTML tags!

@joelostblom 

> I prefer to have it in markdown to be able to export to PDF when needed (even if the box is not included)

When using my Sphinx extension [nbsphinx](https://nbsphinx.readthedocs.io/) the boxes will also be included in PDF output, see e.g. https://nbsphinx.readthedocs.io/_/downloads/en/0.7.1/pdf/#subsection.3.8

This could possibly also added to `nbconvert`'s PDF output?

**Comment by mgeier** [comment]
For future reference: https://github.com/jupyter/notebook/issues/1292

**Comment by itcarroll** [comment]
I thought I had found gold with the discovery of the magic blank line that makes markdown within `<div>` tags render on JLab for these alerts ... and then I found this deficiency in nbconvert. Strongly in support of a fix for this. Apparently a CommonMark solution has still not come about.
